matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/network/NetworkInfoReceiver.kt

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+@file:Suppress("DEPRECATION")
```

Comment:
add a comment explaining why we ignore warning

Reply:
I added the comment, but it is not marked as "outdated". Please see "Files Changed" section.